src/app/docs/input-box/input-box-docs.component.html

```diff
@@ -0,0 +1,232 @@
+            Click
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-click.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Disabled
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-disabled.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Required
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-required.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Error
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-error.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+      </table>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Make input labels succinct and descriptive. Use nouns for input labels. Don't use verbs or questions. For conversational labels or longer descriptions, use section headings outside input boxes. For more information, see the <a href="https://developer.blackbaud.com/skyux/design/guidelines/form-design">form design</a> guidelines.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use section headers outside of input boxes for conversational form design."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/input-box/input-box-content-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Layout"
+    >
+      <div
+        class="sky-docs-text-block"
+      >
+        <sky-docs-design-guideline>
+          <p>
+            Input boxes are most common inside <a href="https://developer.blackbaud.com/skyux/components/modal">modals</a> and follow <a href="https://developer.blackbaud.com/skyux/design/guidelines/form-design">form design</a> guidelines. For two-column or three-column layouts, use the <a href="https://developer.blackbaud.com/skyux/components/fluid-grid">fluid grid</a> narrow spacing option for even vertical and horizontal padding.
```

Comment:
@Blackbaud-AdamFunderburk, I don't see anything about narrow spacing in the fluid grid docs, and I also don't see a narrow option in the spacing docs at https://developer.blackbaud.com/skyux/design/styles/spacing. Am I missing something here? I feel like we're either telling consumers to use something that doesn't exist or that we're referring to it in a way that doesn't match up with our existing docs. 

Also, is `even` the right word at the end? Would `consistent` or `balanced` be better?